retrieval_service/datastore/providers/__init__.py

```diff
@@ -12,6 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from . import cloudsql_postgres, firestore, postgres, spanner_gsql
+from . import cloudsql_postgres, firestore, postgres, spanner
```

Comment:
```suggestion
from . import cloudsql_postgres, firestore, postgres, spanner, spanner_gsql
```

will have to update this at the __ALL__ as well.